packages/nimble-components/src/select/specs/filterable-select-hld.md

```diff
@@ -20,29 +20,65 @@ Our clients have a need of a filterable dropdown component that does not allow a
 
 #### Select
 
-We will provide a means for clients to enable this feature with a new attribute:
+We will provide a means for clients to enable this feature with some new attributes:
 
 ```ts
 export class Select() {
     ...
     @attr({ attribute: 'filter-mode' })
     public filterMode = FilterMode.none;
+
+    /**
+     * Displays a message at the bottom of the dropdown to indicate more options
+     * are currently being loaded.
+     */
+    @attr()
+    public loading-visible = false;
```

Comment:
nit: you should probably be consistent in how technically correct this property/attribute is with `filterMode` above.

```suggestion
    @attr({ attribute: 'loading-visible', mode: 'boolean' })
    public loadingVisible = false;
```